src/no/capraconsulting/buildtools/lifligcdkpipelines/LifligCdkPipelines.groovy

```diff
@@ -286,15 +286,51 @@ def configureVariables(Map config) {
 }
 
 def triggerPipeline(Map config) {
+
   def artifactsBucketName = require(config, "artifactsBucketName")
   def pipelineName = require(config, "pipelineName")
 
-  // Upload a blank file to trigger the event.
-  sh """
-    rm -f /tmp/trigger
-    echo >/tmp/trigger
-    aws s3 cp /tmp/trigger "s3://$artifactsBucketName/pipelines/$pipelineName/trigger"
+  def branchName = env.BRANCH_NAME
```

Comment:
I was thinking that everything associated with the metadata can be added to the try-catch. This allows us to catch errors due to missing environment variables, `git` commands that fail, etc.